notes/stories/film/film-scifi-2010s.st.txt

```diff
@@ -3046,6 +3046,51 @@ unethical business practices [Innovative Online CEO Nolan Sorrento plotted to in
 flying drone [The 101 corporation used flying drones for monitoring people.]
 
 
+movie: Deep Blue Sea 2 (2018)
+=============================
+
+:: Title
+Deep Blue Sea 2
+
+:: Description
+In the film, a shark conservationist is hired to consult on a top-secret
+project involving genetically-enhanced bull sharks funded by a pharmaceutical
+billionaire. However, the highly intelligent super-sharks turn on their
+masters and begin to pick them off one by one. It is a stand-alone sequel to
+the 1999 film Deep Blue Sea.
+
+:: Date
+2018-04-17
+
+:: References
+https://en.wikipedia.org/wiki/Deep_Blue_Sea_2
+
+:: Ratings
+2 <paul>
+
+:: Genre
+sci-fi
+
+:: Choice Themes
+human vs. beast [The people trapped in the sea-based facility were being terrorized by multiple genetically engineered bull sharks.]
+species uplifting [A main novelty of the film is a pharmaceutical billionaire's use of DNA altering drugs to create a number of highly intelligent bull sharks.]
+
+:: Major Themes
+newlywed couple [The newly married neurobiology students Daniel and Leslie Kim were in the honeymoon phase of their relationship.]
+speculative genetic engineering [The pharmaceutical billionaire Carl Durant used DNA altering drugs to genetically modify the brains of five bull sharks to make them smarter.]
+intelligence augmentation technology [The pharmaceutical billionaire Carl Durant used an experimental DNA altering liquid to make himself more intelligent. He used the same drug to increase the shark Bella's intelligence by a thousand fold.]
+employer and employee [The pharmaceutical billionaire Carl Durant directly oversaw the work the specialists that he'd brought in to work at his sea-based facility. forced his employee Trent to sign what was in Trent's view an onerous nondisclosure agreements in an effort to protect his trade secrets.]
```

Comment:
something is wrong here

Reply:
Fixed **employer and employee** motivation.

And yes, I agree with you about the major theme comment motivations. In this particular instance, upon sober reflection a day later, I've knocked down both **newlywed couple** and **employer and employee** to minor.

The only thing I would add is we use that we strive for that same standard with the interpersonal relationships in APH and other such series.